.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Mind removing this file from the PR?

Reply:
shouldn't be in here... sorry, please remove